lavamoat/build-system/policy.json

```diff
@@ -1225,14 +1181,21 @@
         "assert": true,
         "path": true
       },
+      "globals": {
+        "afterAll": true,
+        "process.cwd": true
```

Comment:
these were not introduced here, looks like our improved policy generation found them this time.